.github/workflows/e2ev2-provision-test.yaml

```diff
@@ -91,7 +91,7 @@ jobs:
 
       - name: Print test logs
         shell: bash
-        run: cat testing/e2e/job-app-routing-operator-e2e.log
+        run: head -n 100 testing/e2e/job-app-routing-operator-e2e.log && tail -n 200 testing/e2e/job-app-routing-operator-e2e.log
```

Comment:
print that we are printing the first 100 and last 200 logs so no one gets confused in the future that the middle is missing

Reply:
done